### PR TITLE
fix: STORAGE_METHOD -> STORAGE_TYPE

### PR DESCRIPTION
**Pull request by jcchavezs**
From https://github.com/openzipkin/zipkin/pull/3415#discussion_r791507531

### CODE REVIEW

**Review by jonkerj — CHANGES_REQUESTED** [review]
You need to bump the version (just increase the patch level) in `Chart.yaml` as well

**Comment by jonkerj** [comment]
I really like the way this is going. Don't forget to also update the chart version (`.version`).

File: charts/zipkin/README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+```console
+helm repo add openzipkin https://openzipkin.github.io/zipkin
+```
+
+You can then run `helm search repo zipkin` to see the charts.
```
Comment (jcchavezs) [inline charts/zipkin/README.md]:
```suggestion
You can then run `helm search repo openzipkin` to see the charts.
```

**Comment by jcchavezs** [comment]
> You need to bump the version (just increase the patch level) in Chart.yaml as well

I will do a chart release after merge.

File: .github/workflows/helm_release.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# yamllint --format github .github/workflows/helm_release.yml
+---
+name: helm_release
+
+# We release helm_charts into https://openzipkin.github.com/index.yaml
```
Comment (jcchavezs) [inline .github/workflows/helm_release.yml]:
```suggestion
# We release helm_charts into https://github.com/openzipkin/zipkin/blob/gh-pages/index.yaml
```

**Pull request merged**